src/_guides/libraries/create-library-packages.md

```diff
@@ -205,7 +205,7 @@ or any `packages` directories. For more information, see
 
 {% comment %}
 PENDING: here just to make it easy to find discussions of `packages`...
-{% include coming-release.html %}
+{% include packages-dir.html %}
```

Comment:
The actual packages-dir.html include (when not commented out) produces a standard note about packages/.packages. E.g. see https://kw-www-dartlang-1.firebaseapp.com/guides/libraries/private-files#packages-and-packages.